sys/include/net/ng_netif.h

```diff
@@ -90,6 +90,41 @@ void ng_netif_remove(kernel_pid_t pid);
  */
 kernel_pid_t *ng_netif_get(size_t *size);
 
+/**
+ * @brief   Converts a hardware address to a human readable format.
```

Comment:
This sounds misleading: a `'\0'` terminated string can also just be the bytes of the address (not converted to hex strings!) + an appended `'\0'`